src/components/notification/bl-notification.css

```diff
@@ -32,7 +32,7 @@
   touch-action: none;
 }
 
-:host([dir="rtl"]) .notification {
+:host(:has([dir="rtl"])) .notification {
```

Comment:
is it working?

Reply:
It works for notification, but not for other components unfortunately 😞